jujugui/static/gui/src/app/components/deployment-flow/signup/signup.js

```diff
@@ -51,6 +51,7 @@ YUI.add('deployment-signup', function() {
       @method _handleSignup
     */
     _handleSignup: function() {
+      document.cookie = 'signup-seen=true';
```

Comment:
We probably want to set a cookie sometime far in the future - I believe this causes it to expire at the end of the browsing session.